pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -164,32 +165,53 @@ class EmuFreeBackend(PasqalEmulator):
     default_config = DEFAULT_CONFIG_EMU_FREE
 
 
-class EmuMPSBackend(RemoteBackend):
+class RemoteEmulatorBackend(RemoteBackend, EmulatorBackend):
+    _device_type: DeviceTypeName
```

Comment:
Don't you have to type this as a class var?
```suggestion
    _device_type: ClassVar[DeviceTypeName]
```